stellargraph/core/element_data.py

```diff
@@ -107,6 +107,8 @@ class ElementData:
         shared (dict of type name to pandas DataFrame): information for the elements of each type
     """
 
+    _SHARED_REQUIRED_COLUMNS = []
```

Comment:
I know this is called "shared" but possibly an extra comment to say that it's sometimes overridden in derived classes?